backend/data.go

```diff
@@ -8,7 +8,7 @@ import (
 	"time"
 
 	"github.com/grafana/grafana-plugin-sdk-go/data"
-	jsoniter "github.com/json-iterator/go"
+	sdkjsoniter "github.com/grafana/grafana-plugin-sdk-go/data/utils/jsoniter"
```

Comment:
nit: I would just call use it as `jsoniter` if you are only using one.

Reply:
I want to make it explicit.